README.rst

```diff
@@ -269,7 +282,7 @@ Then Lightning-GPU with MPI support can then be installed with ``pip``:
 .. code-block:: console
 
     PL_BACKEND="lightning_gpu" python scripts/configure_pyproject_toml.py
-    CMAKE_ARGS="-DENABLE_MPI=ON" python -m pip install -e .
+    CMAKE_ARGS="-DENABLE_MPI=ON" python -m pip install -e . -vv
```

Comment:
Not sure if we need to add the verbosity flag in the installation guideline. We can let developers to choose these custom installation flags. What do you think?
```suggestion
    CMAKE_ARGS="-DENABLE_MPI=ON" python -m pip install -e .
```

Reply:
I think that it is better to add the flag for beginner developers. For example, in case something goes wrong with the installation process, they will immediately notice it.